src/domain/changeSecretNumber.ts

```diff
@@ -0,0 +1,8 @@
+export const changeSecretNumber = (number: string) => {
+  const secretNumber = number
+    .split('')
+    .map((data) => data.replace(data, '*'))
+    .join('')
+
+  return secretNumber
+}
```

Comment:
이건 취향이 조금 담겨있는데 바로 return해도 좋을거 같네요
```suggestion
export const changeSecretNumber = (number: string) => {
  return number
    .split('')
    .map((data) => data.replace(data, '*'))
    .join('')
}
```

Reply:
그리고 parameter의 이름이 `number`인데 string 타입으로 받는건 조금 어색하게 느껴지는데요.

parameter의 이름도 보다 적절하게 네이밍 해주시면 좋을거 같아요